src/CrateTokenV1.sol

```diff
@@ -199,31 +210,11 @@ contract CrateTokenV1 is ERC20Upgradeable, ReentrancyGuard {
             minTokens, // amountTokenMin
             amountETHDesired, // amountETHMin
             address(0), //where to send LP tokens
-            block.timestamp + 300 // Deadline (current time plus 300 seconds)
+            block.timestamp // Deadline (current time)
         );
-        require(amountToken > 0 && amountETH > 0 && liquidity > 0, "Liquidity addition failed.");
+        if (amountToken == 0 || amountETH == 0 || liquidity == 0) revert Zero();
         (bool protocolFeePaid,) = protocolFeeDestination.call{value: 0.3 ether}("");
-        require(protocolFeePaid, "Failed to pay protocol fee");
-
+        if (!protocolFeePaid) revert TransferFailed();
         emit LiquidityAdded(amountToken, amountETH, liquidity);
     }
-
-    function _update(address from, address to, uint256 value) internal override {
```

Comment:
Why was this removed?

Reply:
achieved nothing

Reply:
Seemed unnecessary given checks in other functions, am happy to add re-add though